knative/docs
# Update docs repo for 0.26

Fixes #4275 

## Proposed Changes 

- Release the 0.26 version of the docs.

/assign @julz

to verify technically

/remove-approve

Since I want someone else to actually make sure I did the right thing by the site.
